README.md

```diff
@@ -83,5 +83,30 @@ mypy --strict --install-types --show-error-codes minique
 REDIS_URL=redis://localhost:6379/0 pytest .
 ```
 
+### Release
+
+```shell
+# decide on a new versio number and set it
+vim minique/__init__.py
+__version__ = "0.9.0"
```

Comment:
Also: `hatch version minor` (and friends) :)

Reply:
`Command 'hatch' not found` :disappointed: